domain/src/main/java/org/oppia/domain/oppialogger/loguploader/LogUploadWorker.kt

```diff
@@ -0,0 +1,110 @@
+package org.oppia.domain.oppialogger.loguploader
+
+import android.content.Context
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import org.oppia.domain.oppialogger.analytics.AnalyticsController
+import org.oppia.domain.oppialogger.exceptions.ExceptionsController
+import org.oppia.domain.oppialogger.exceptions.toException
+import org.oppia.util.data.DataProviders
+import org.oppia.util.logging.ConsoleLogger
+import org.oppia.util.logging.EventLogger
+import org.oppia.util.logging.ExceptionLogger
+import javax.inject.Inject
+
+const val LOG_UPLOAD_WORKER = "log_upload_worker"
```

Comment:
Suggestion: 
`const val TAG = "LogUploadWorker.tag"`

Shift this to companion object and use it like this:
`LogUploadWorker.TAG ` everywhere outside this file.

Reply:
I don't actually need it anywhere outside this file as of now but moved it to companion object as we may use it in the future while logging work requests.

Reply:
Okat